smartsim/_core/utils/redis.py

```diff
@@ -241,7 +241,8 @@ def shutdown_db_node(host_ip: str, port: int) -> t.Tuple[int, str, str]:  # cov-
 
     if returncode != 0:
         logger.error(out)
-        logger.error(err)
+        if err:
+            logger.error(err)
```

Comment:
if the RC is non-zero, we shouldn't care about the error message to log... that just hides that the error occurred.

i think you might just get rid of `if err` to make sure we don't hide a problem:

`logger.error("something happened. rc: {returncode}, err: {err}")`

Reply:
Thanks, done as part of the other PR.